omod/src/main/webapp/resources/scripts/navigator/navigatorHandlers.js

```diff
@@ -134,7 +135,7 @@ function QuestionsKeyboardHandler(questionModels) {
     api.selectedQuestion = function() {
         return selectedModel(questions);
     };
-    api.handleUpKey = function() {
+    api.prevQuestion = function() {
```

Comment:
renamed "handleUpKey" and "handleDownKey" to more "functional" names since we are now triggering them outside of that context